cookbook/frontend/assetic_php.rst

```diff
@@ -0,0 +1,152 @@
+.. index::
+    single: Front-end; Assetic, Bootstrap
+
+Combining, Compiling and Minimizing Web Assets with PHP Libraries
+=================================================================
+
+The official Symfony Best Practices recommend to use Assetic to
+:doc:`manage web assets </best_practices/web-assets>`, unless you are
+comfortable with JavaScript-based frontend tools.
+
+Even if those JavaScript-based solutions are the most suitable ones from a
+technical point of view, using pure PHP alternative libraries can be useful in
+some scenarios:
+
+* If you can't install or use ``npm`` and the other JavaScript solutions;
+* If you prefer to limit the amount of different technologies used in your
+  applications;
+* If you want to simplify application deployment.
+
+In this article you'll learn how to combine and minimize CSS and JavaScript files
```

Comment:
It's to allow this kind of crazy word order in the sentence. Without comma, you would do "You'll learn how to combine and minimize CSS and JS files in this article" (time and location are always located at the end of the sentence). This is perfectly fine when you're speaking to someone, but when reading, it's common to put the location in front of the sentence. When doing that, you must add a comma before you start the sentence: "In this article, you'll learn [...]"